### DISCUSSION TITLE
Test cases

### DISCUSSION THREAD

**Discussion by seohyeon0322** [ANLAB-KAIST/KENSv3#89] [Project Q&A 3. Internet Layer (IP/Routing)]
I can't understand how test case check each node's cost information. 
I thought we just update distance vector table in our code (We made our own distance vector tables.)
Do we have to do something more than updating our distance vector node?

Also, according to documents, when the node receives response, it updates its distance vector table. After updating, do we have to do something more ..? I thought we have to send more packets, but documents doesn't say anything after updating distance vector table.
The example cap files have a lot of packets, however it seems that there is no packet exchange after updating distance vector table according to document.

**Comment by Pusnow** [comment]
You have to implement `ripQuery()`. The grader will use this method for grading.

https://github.com/ANLAB-KAIST/KENSv3/blob/22b8748332f44905abb3277dad14fcc088bc36c6/app/routing/RoutingAssignment.cpp#L27-L37
